presto-raptor-legacy/src/main/java/io/prestosql/plugin/raptor/legacy/storage/ShardStats.java

```diff
@@ -87,6 +87,7 @@ private static ColumnStats doComputeColumnStats(OrcReader orcReader, long column
         if (type.equals(BigintType.BIGINT) ||
                 type.equals(DateType.DATE) ||
                 type.equals(TimestampType.TIMESTAMP)) {
+            // TODO: timestamp
```

Comment:
Not sure what we want to do here.

Reply:
Since we're now storing micros, there's nothing to do here, as the stats will also be in micros.

As a follow up, or as part of this PR, we could change Raptor to `timestamp(6)` and the only difference would be that users get more precision.